pkg/webhooks/clusterqueue_webhook_test.go

```diff
@@ -156,7 +157,7 @@ func TestValidateClusterQueue(t *testing.T) {
 					*testingutil.MakeFlavorQuotas("x86").Resource("cpu", "1", "", "1").Obj()).
 				Obj(),
 			wantErr: field.ErrorList{
-				field.Invalid(resourceGroupsPath.Index(0).Child("flavors").Index(0).Child("resources").Index(0).Child("lendingLimit"), "1", limitIsEmptyErrorMsg),
+				field.Invalid(resourceGroupsPath.Index(0).Child("flavors").Index(0).Child("resources").Index(0).Child("lendingLimit"), "1", fmt.Sprintf(limitIsEmptyErrorMsgTemplate, "cohort")),
```

Comment:
please write literal string in test

Reply:
wrote the literal string, but it's is ignored when running tests:
https://github.com/kubernetes-sigs/kueue/blob/2b92883a0ec779ef0cd38b369b316fa0828bfcb4/pkg/webhooks/clusterqueue_webhook_test.go#L335-L336

I can try to modify the logic so that error strings are actually compared.

Reply:
However, it's ignored in other tests as well, and it makes sense to not compare error detail messages in most of the cases.  Maybe we can make an exception just for this particular subtest.